lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/gateway/ThirdPartyDocumentGateway.java

```diff
@@ -165,12 +209,8 @@ private DocumentCheckResult responseHandler(CloseableHttpResponse httpResponse)
         LOGGER.info("Third party response {}", responseBody);
 
         if (statusCode == 200) {
-            // HttpEntity entity = httpResponse.getEntity();
-            // String responseBody = EntityUtils.toString(entity);
-
             DcsResponse unwrappedDcsResponse =
                     dcsCryptographyService.unwrapDcsResponse(responseBody);
-            validateDcsResponse(unwrappedDcsResponse);
```

Comment:
I think it would be worth keeping this for logging just remove the exception

Reply:
Re-added and removed exception, however I changed the log line level from error to info, as this is expected to happen on every form retry